src/Modules/AuthModule/components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,6 +1,251 @@
+                        type="button"
+                        onClick={togglePasswordVisibility2}
+                        title=" btn"
+                      >
+                        <i
+                          className={`fa-regular fa-eye${
+                            !showNewPassword ? "-slash" : ""
+                          }`}
+                        ></i>
+                      </button>
+                      {/* <div className="border_bottom"></div> */}
+                    </div>
+                    {errors.newPassword && (
+                      <div className="alert alert-warning p-1 mx-2">
+                        {errors.newPassword.message}
+                      </div>
+                    )}
+                    {/* Confirm New Password  */}
+                    <div className="visibilty-password input m-3">
+                      <label
+                        className="color-text "
+                        htmlFor="exampleFormControlInput2"
+                      >
+                        Confirm New Password
+                      </label>
+                      <input
+                        type={showConfirmPassword ? "text" : "password"}
+                        className="form-control shadow-none"
+                        style={{
+                          backgroundColor: showConfirmPassword
+                            ? "transparent"
+                            : "#fff",
+                          borderTop: showConfirmPassword ? "none" : "",
+                          borderRight: showConfirmPassword ? "none" : "",
+                          borderLeft: showConfirmPassword ? "none" : "",
+                          borderRadius: showConfirmPassword ? "0px" : "",
+                          color: showConfirmPassword  ? "white" :  "",
+                        }}
+                        placeholder="Confirm New Password"
+                        {...register("confirmNewPassword", {
+                          required: "confirmNewPassword is required ",
+                          validate: (value) =>
+                            value === watch("newPassword") ||
+                            "the passwords dont match ",
+                          pattern: {
+                            value:
+                              /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[a-zA-Z]).{8,}$/,
+                            message:
+                              "Password must contain at least 8 characters, including upper and lowercase letters, and numbers",
+                          },
+                        })}
+                      />
+                      {/* button eye password */}
+                      <button
+                        className="btn btn-outline-secondary for-visibilty-password-button"
+                        type="button"
+                        onClick={togglePasswordVisibility3}
+                        title=" btn"
+                      >
+                        <i
```

Comment:
I recommend using SVG icon library something like Lucide React, it provides an extensive icon set.
The problem with font icons is that it will be read as garbage characters to screen readers so we need to hide it from screen readers and add an aria label describing what this icon represents, with SVG we only need to provide an aria label for description only.